lib/gooddata/lcm/actions/synchronize_clients.rb

```diff
@@ -87,6 +88,21 @@ def call(params)
                    "Details: #{sync_result['links']['details']}")
             end
 
+            # rubocop:disable Style/RescueStandardError
+            if keep_only_previous_masters_count >= 0
+              number_of_deleted_projects = master_projects.count - (keep_only_previous_masters_count + 1)
+
+              if number_of_deleted_projects.positive?
```

Comment:
why don't we just simply check something like: master_projects.count - keep_only_previous_masters_count > 1

Reply:
discussed about it and we keep it as current because we need to remove latest master workspace from list